core/command_test.go

```diff
@@ -6,13 +6,8 @@ import (
 )
 
 func TestStatePath(t *testing.T) {
-	subTree := &File{"c", nil, 50, true, []*File{
-		&File{"d", nil, 50, false, []*File{}},
-	}}
-	tree := &File{"a", nil, 50, true, []*File{
-		&File{"b", nil, 50, false, []*File{}},
-		subTree,
-	}}
+	tree := TstFolder("a", TstFile("b", 50), TstFolder("c", TstFile("d", 50)))
```

Comment:
Agree 👍 I was trying to have something really concise but I guess the outcome is less readable.. Will fix